esp/src/eclwatch/HPCCPlatformWidget.js

```diff
@@ -162,6 +163,27 @@ define([
             /* */
             var teaserNode = dom.byId(this.id + "teaser");
             srcReact.render(ComingSoonModule.ComingSoon, { style: { color: "white" } }, teaserNode);
+
+            const DAY = 1000 * 60 * 60 * 24;
+            platformModule.fetchCheckFeatures().then(function (features) {
+                const age = Math.floor((Date.now() - new Date(features.BuildTagTimestamp).getTime()) / DAY);
```

Comment:
If features didn't come back for some reason, might have an invalid date here also, and a NaN for age.